app/views/admin/organisations/index.html.erb

```diff
@@ -1,7 +1,31 @@
-<h1 class="govuk-heading-l">All organisations</h1>
-<h3 class="govuk-heading-m">
-  <%= pluralize(@organisations.count, 'organisation') %> are signed up to GovWifi
-</h3>
+<h1 class="govuk-heading-l">Summary of all organisations</h1>
+<div class="govuk-grid-column admin-panel-stats-container">
+  <div class="govuk-grid-column-one-third stats-align-center">
+    <div class="govuk-heading-l">
+    <strong class="total-org-stat-results-title"><%= @total_number_of_locations%></strong>
```

Comment:
Closing tag whitespace is off, not sure why rubocop isn't picking up on this.
Have added to tech debt board to check